src/DynamoPackages/PackageManagerExtension.cs

```diff
@@ -0,0 +1,145 @@
+﻿using Dynamo.Core;
+using Dynamo.DSEngine;
+using Dynamo.Extensions;
+using Dynamo.Interfaces;
+using Dynamo.Models;
+using Dynamo.Utilities;
+using Greg;
+using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Dynamo.PackageManager
+{
+    public class PackageManagerExtension : IExtension
+    {
+        public string Name { get { return "DynamoPackageManager"; } }
```

Comment:
Let's move all properties and fields to the top of the file and encapsulate in a `region`.